mimic/test/dummy.py

```diff
@@ -93,36 +96,103 @@ def resource(self):
+        :param boolean enabled: whether or not the service is enabled
+            for all users. Services can be disabled for all tenants but still
+            be enabled on a per-tenant basis.
+        :param text_type uuid: unique ID for the endpoint within the service.
+        :param text_type tenantid_alias: by default the system uses the text
+            '%tenant_id%' for what to replace in the URLs with the tenantid.
+            This value allows the service adminstrator to use a different
+            textual value. Note: This is not presently used by Mimic which
+            just appends the tenant-id for internally hosted services, and
+            simply uses the URLs as is for externally hosted services.
+        """
+        self.id_key = uuid
+        self.region_key = region
+        self.type_key = type_id
         self.name_key = name
-        self.regions_and_versions = regions_and_versions
-
-    def catalog_entries(self, tenant_id):
-        """
-        List catalog entries for the Example API.
-        """
-        endpoints = [Endpoint(tenant_id, each[0], 'uuid', each[1]) for each in self.regions_and_versions]
-        return [Entry(tenant_id, "externalServiceType", "externalServiceName", endpoints)]
-
-    def uri_for_service(self, region, service_id):
-        """
-        Return the URI for the service in the given region.
-        """
-        for r, u in [(each[0], each[2]) for each in self.regions_and_versions]:
-            if r == region:
-                return u
-        return u""
+        self.enabled_key = enabled
+        self.publicURL = publicURL if publicURL is not None else url
+        self.internalURL = internalURL if internalURL is not None else url
+        self.adminURL = adminURL if adminURL is not None else url
+        self.tenantAlias = tenantid_alias
+        self.versionId = version
+        self.versionInfo = (versionInfoURL
+                            if versionInfoURL is not None
+                            else url + '/versionInfo')
+        self.versionList = (versionListURL
+                            if versionListURL is not None
+                            else url + '/versions')
+
+
+def make_example_external_api(name=u"example",
+                              endpoint_templates=[ExampleEndpointTemplate()],
+                              set_enabled=None):
+    """
+    Initialize an :obj:`ExternalApiStore` for a given name.
+
+    :param text_type name: user-visible name of the service.
+    :param list endpoint_templates: list of endpoint templates to
+        initialize the API store with.
+    :param boolean or None set_enabled: If none, the endpoint templates
+        are used AS-IS. If a boolean type, then it sets all the templates
+        to have the same default accessibility for all tenants.
+
+    Note: The service-type of the first endpoint template is used as the
+        service type for the entire Api Store, and is enforced that all
```

Comment:
Is `Api Store` capitalized here to reference the `ExternalApiStore` object? If so, it might be clearer to use :obj:`ExternalApiStore`. :-)